x/lightclient/ante/ibc_msg_update_client.go

```diff
@@ -14,6 +14,19 @@ import (
 	"github.com/dymensionxyz/gerr-cosmos/gerrc"
 )
 
+var (
+	blockUrlUpdate      = sdk.MsgTypeURL(&ibcclienttypes.MsgUpdateClient{})
+	blockUrlMisbehavior = sdk.MsgTypeURL(&ibcclienttypes.MsgSubmitMisbehaviour{})
+)
+
+// Depth is the nesting depth of the message with authz and gov proposal. depth 0 is a top level message.
+// Only blanket rejects depth greater than zero because we have our own custom logic for depth 0
+// Note that there is never a genuine reason to pass both ibc update client and misbehaviour submission through gov or auth,
+// it's always done by relayers directly.
+func BlockMsg(typeURL string, depth int) bool {
```

Comment:
I think we can move this code to the rejector file as well.
can be named better (e.g `BlockEncapsulatedMsg`), as the difference with `BlockTypeUrls` is the depth > 0 condition

Reply:
done 👍